pkg/dockerutil/dockerutils.go

```diff
@@ -331,6 +332,53 @@ func CheckDockerVersion(versionConstraint string) error {
 	return nil
 }
 
+// CheckDockerCompose determines if docker-compose is present and executable on the host system. This
+// relies on docker-compose being somewhere in the user's $PATH.
+func CheckDockerCompose(versionConstraint string) error {
+	executableName := "docker-compose"
+	if runtime.GOOS == "Windows" {
+		executableName = "docker-compose.exe"
+	}
+
+	path, err := exec.LookPath(executableName)
+	if err != nil {
+		return fmt.Errorf("no docker-compose")
+	}
+
+	out, err := exec.Command(path, "version", "--short").Output()
+	if err != nil {
+		return err
+	}
+
+	version := string(out)
+	version = strings.TrimSuffix(version, "\n")
+
+	dockerComposeVersion, err := semver.NewVersion(version)
```

Comment:
Should all this stuff be consolidated between the docker and docker-compose checks? Maybe not for just the first copy of everything, but we shouldn't continue the pattern of copying after this if it happens again. If you prefer to consolidate the docker and docker-compose checks it would be OK by me.